web/web_test.go

```diff
@@ -1402,6 +1403,406 @@ func TestDropForeignKey(t *testing.T) {
 	}
 }
 
+func TestRenameIndexes(t *testing.T) {
+	tc := []struct {
```

Comment:
I think we are missing a few test cases:

"Test rename multiple indexes",
"New name conflicts with an existing table",
"New name conflicts with an existing index",
"New name conflicts with an existing foreign key",
"Conflicts within new name array"

Reply:
added pl